frog/imports/client/Wiki/index.js

```diff
@@ -140,9 +143,10 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
 
     window.wikiDoc = this.wikiDoc;
     if (!this.props.embed) {
-      Mousetrap.bindGlobal('ctrl+n', () =>
-        this.setState({ createModalOpen: true })
-      );
+      Mousetrap.bindGlobal('ctrl+n', () => {
+        if (!this.state.settings?.limitPageCreation && !this.state.isOwner)
```

Comment:
So, we allow page creation if:
- `settings` is undefined (please consider giving settings a default value to simply the logic here), OR
- `limitPageCreation` is false, meaning page creation is allowed (consider changing limitPageCreation to allowPageCreation to remove the ! and improve readability, as per http://davej.io/2018/05/negative-naming.html), AND
- `isOwner` is false, **meaning that the owner of the wiki cannot add pages?** Make sure this is intended.